.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,11 +4,10 @@ Please fill in the following checklist, removing items that do not apply.
 See also https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md
 -->
 
-- [ ] The page (if new), does not already exist in the repository.
-- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
-- [ ] The page has 8 or fewer examples.
-- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
-- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
-- [ ] The page description includes a link to documentation or a homepage (if applicable).
+- **Version of the command being documented (if known):**  
 
-**Version of the command being documented (if known):**
+- [ ] All command page(s) are in correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`.
```

Comment:
This appears to be an exhaustive list without the `etc` at the end there, so we'll need to remember to update this when adding a new platform - which I'm not a fan of.